microsoft-edge/extensions-chromium/publish/api/using-addons-api.md

```diff
@@ -237,6 +245,15 @@ Header Parameters: Authorization: Bearer $TOKEN
 https://api.addons.microsoftedge.microsoft.com/v1/products/$productID/submissions/operations/{operationID}
 ```
 
+
+## Getting started with v2
```

Comment:
are lines 249-255 a typo (251: no start of sentence)?  
are they placed in wrong spot at bottom of article?  
Why indented via > as if an alert?  
Why placed within h4 section - above its minor See also line - instead of below that section (just above big h2 See also section) or at top of article?

Files Changed view after expanding lines above & below:

Reply:
i moved section to near top of article.  snapshot: 
```
<!-- ====================================================================== -->
## Getting started with v2 of this REST API

To use v2 of this REST API:

1. At Partner Center, opt-in to the new API key management experience.

1. Re-generate your ClientId and secrets.<!-- todo: reword, b/c secrets are no longer used, according to "v2 of this REST API uses API keys, whereas v1 used Secrets instead" -->  You might need to update your authentication workflows.

1. Re-configure any existing Continuous Integration (CI) or Continuous Development (CD)<!-- todo: does CD stand for Continuous Development? --> pipelines that might be impacted by the changes to the access token URL and API key.
```

Reply:
further improved new h2 section "Version 2 of the REST Publish API"